### ISSUE TITLE
Embed block: Clicking on Watch on YouTube causes an error on Chrome

### ISSUE THREAD

**Issue by aisajib** [Automattic/wp-calypso#77056]
### Quick summary

A user reported, and we were able to replicate, that when you click "Watch on YouTube" on an embedded YouTube video on WordPress.com sites using Chrome, you get an error message that says www.youtube.com is blocked. 

However, visiting www.youtube.com directly works just fine. It works on other browsers as well (tested on Safari, Firefox, and Edge). Only Chrome appears to be having this issue. 

### Steps to reproduce

1. Open a WordPress.com post with a YouTube video embedded or publish a new post with a YouTube video embedded.
2. Visit the post using Google Chrome. 
3. Click on "Watch on YouTube."

### What you expected to happen

The video should open on YouTube.com without any issues. 

### What actually happened

On Google Chrome, we get the following error: 

<img width="507" alt="CleanShot 2023-05-18 at 10 09 15@2x" src="https://github.com/Automattic/wp-calypso/assets/2541844/28a0fff0-ad2d-4d21-991c-9e78eee0a0d7">

[Here's a sample post to try this one](https://wptutos.wordpress.com/2022/08/08/yt/).

### Impact

Most (> 50%)

### Available workarounds?

Yes, easy to implement

### Platform (Simple and/or Atomic)

Simple, Atomic

### Logs or notes

Replicated on Google Chrome Version 113.0.5672.126 (Official Build) (arm64)
Replicated on WordPress.com Simple and Atomic, but not on self-hosted sites. 
Reported on the [WordPress.com forum](https://wordpress.com/forums/topic/watch-on-youtube-link-not-working/?view=all).

**Comment by aisajib** [comment]
Although the self-hosted site I tested on didn't reproduce this issue, @druesome  ran this on his self-hosted site, and we were able to reproduce it there.

**Comment by simison** [comment]
Tested:

- On Chrome browser where I'm logged in with YouTube, everything works as expected. The "Watch on YouTube" opens to a new tab, play button plays the video, etc.

    <img width="500" alt="Screenshot 2023-05-18 at 09 38 55" src="https://github.com/Automattic/wp-calypso/assets/87168/b68ec297-252b-4a03-893c-1aceda774064">

    <img width="500" alt="Screenshot 2023-05-18 at 09 39 33" src="https://github.com/Automattic/wp-calypso/assets/87168/2150faae-ee83-4870-b161-8f27752ad497">

- On incognito Chrome (so no 3rd party cookies, not logged in to YouTube) I get the error page in new tab:

    <img width="500" alt="image" src="https://github.com/Automattic/wp-calypso/assets/87168/6ffe1e96-b637-4f2c-9ffd-a51189fd3d31">

- On the same incognito Chrome, playing the video in embed works just fine.

- On the same incognito if I then just visit youtube.com and accept cookies, the clicking the embed or refreshing the new tab opened from embed now works fine.

    <img width="500" alt="Screenshot 2023-05-18 at 09 46 23" src="https://github.com/Automattic/wp-calypso/assets/87168/d8e0fa73-3c36-43ae-a90d-f5d5c141f6ac">

- On Firefox incognito, not logged in to YouTube, I get the cookie banner directly in the new tab and everything works:

    <img width="500" alt="Screenshot 2023-05-18 at 09 43 10" src="https://github.com/Automattic/wp-calypso/assets/87168/2e877469-44c6-42ad-ba1b-6629a21147f3">

- I tested with another video, same results.

**Comment by cuemarie** [comment]
📌 SCRUBBING : RESULT - *Replicated / Could Not Replicate / Uncertain*
- Tested on Simple – **Replicated**
- Replicable outside of Dotcom – Yes/No

📌 FINDINGS/SCREENSHOTS/VIDEO
- Replicated using the example post linked in the original report, on `Chrome Version 113.0.5672.126 (Official Build) (x86_64)` 
- The issue occurs consistently in Chrome when third party cookies are blocked. My logged in state with YT doesn't seem to matter.

#### Screenshots/Recordings

**Third Party Cookies Blocked**

https://github.com/Automattic/wp-calypso/assets/27249804/3ca8b170-3758-4d5d-b112-ac7f4a370b2e

**All Cookies Allowed**

https://github.com/Automattic/wp-calypso/assets/27249804/078eb6a7-3195-4487-82a4-3c971be6ff97

📌 ACTIONS
- Scrubbed, awaiting assignment

**Comment by cuemarie** [comment]
Escalation thread: p1684390423501289-slack-C02FMH4G8

Signs indicate this is a change on Youtube's end related to third-party cookies.

**Comment by wynwin** [comment]
Another report in 6399151-zen

**Comment by filipanoscampos** [comment]
6454295-zd-woothemes

**Comment by Jaccuse** [comment]
6543995-zen

**Comment by jp-imagines** [comment]
Another report in 6634135-zd-a8c.

Possibly noteworthy (at least as workarounds): you can ctrl+click (or right click > open in new tab) on the video's title or Watch on YouTube links, and the video will then open properly. Hyperlinked text isn't affected either.

**Comment by nazilya06** [comment]
7127875-zen

**Comment by cuemarie** [comment]
@ahegyes  reported this in https://github.com/Automattic/jetpack/issues/33580 , with the following additional notes that may be helpful!

> YouTube has started introducing a new COOP policy that is affecting YouTube embeds when Jetpack's shortcode embeds module is activated. Namely, it prevents the video from opening in a new tab when clicking the title or the Watch on YouTube banner.
>
>This is because of the sandbox parameter added by the module at this line:https://github.com/Automattic/jetpack/blob/f90120b19e02f2b3a806a7c3e8c8a21c75b6b2b0/projects/plugins/jetpack/modules/shortcodes/youtube.php#L345
>
>I confirmed that the error goes away if the `sandbox` attribute is either not present or if the list includes the `allow-popups-to-escape-sandbox` value, [as suggested by this comment](https://bugs.chromium.org/p/chromium/issues/detail?id=1463173#c13).

---

📌 REPRODUCTION RESULTS
- Retested on Atomic today, and can reproduce the `ERR_BLOCKED_BY_RESPONSE` warning even with all third party cookies allowed.
- Other browsers, such as Firefox, still don't encounter the error.

📌 ACTIONS
- updated title
- CC: @jeherve is this something Jetpack could take a look into?

**Comment by jeherve** [comment]
> is this something Jetpack could take a look into?

Yes, since this is an issue with the Youtube shortcode, it should really live in the Jetpack repo. I'll see if we can take a look at this.
